comfy_cli/cql/annotations_source.py

```
@@ -0,0 +1,147 @@
+
+    # Try the network when allowed and either forced or the cache is stale/missing.
+    if not _network_disabled():
+        try:
+            data = _fetch(filename)
+            try:
+                cache.parent.mkdir(parents=True, exist_ok=True)
+                cache.write_bytes(data)
+            except OSError:
+                pass  # cache write is best-effort
+            return data
+        except Exception:
```

Pylint (4.0.6)

[warning] Catching too general exception Exception

(W0718)